core/mondoo-windows-security.mql.yaml

```diff
@@ -1032,8 +968,8 @@ queries:
             default: "Failure";
           }
     mql: |
-      auditpol.where(subcategory == props.auditpolFileShare).list != []
-      auditpol.where(subcategory == props.auditpolFileShare).all(inclusionsetting == props.auditpolSuccessFailure)
+      auditpol.where(subcategoryguid == "0CCE9244-69AE-11D9-BED3-505054503030").list != []
+      auditpol.where(subcategoryguid == "0CCE9244-69AE-11D9-BED3-505054503030").all(inclusionsetting == props.auditpolSuccessFailure)
```

Comment:
the guid is wrong. it should be 0CCE922F-69AE-11D9-BED3-505054503030

Reply:
Fixed in latest commit

Reply:
Fixed in latest commit